--- origsrc/icu/source/config/mh-cygwin	2007-09-13 18:18:10.000000000 -0500
+++ src/icu/source/config/mh-cygwin	2007-11-23 01:02:21.024875000 -0600
@@ -46,10 +46,6 @@
 STATIC_O = ao
 LIBICU = cyg$(ICUPREFIX)
 
-# Change the stubnames so that poorly working FAT disks and installation programs can work.
-# This is also for backwards compatibility.
-DATA_STUBNAME = dt
-I18N_STUBNAME = in
 
 ## Compilation rules
 %.$(STATIC_O): $(srcdir)/%.c
